pyccel/stdlib/math/pyc_math.f90

```diff
@@ -0,0 +1,150 @@
+module pyc_math
+
+use ISO_C_BINDING
+
+implicit none
+
+real(C_DOUBLE), parameter, private :: pi = 4.D0 * DATAN(1.D0)
```

Comment:
You should also use `ISO_C_BINDING` on literals to avoid potential casting. I.e:
```
4.0_C_DOUBLE
```